index.js

```diff
@@ -39,9 +44,11 @@ const destroyCircular = (from, seen) => {
 	return to;
 };
 
-const serializeError = value => {
+const serializeError = (value, options = {}) => {
+	const {maxDepth = 0} = options;
```

Comment:
Is `0` the best default? Just asking as I'm not really sure what the best default would be. (It would also be a breaking change).

Reply:
You're right, I don't think we should make any breaking changes — I'll default it to `Infinity` for now.